src/visual/seriesColor.js

```diff
@@ -48,9 +52,13 @@ export default {
             var dataEach = function (data, idx) {
                 var itemModel = data.getItemModel(idx);
                 var color = itemModel.get(colorAccessPath, true);
+                var borderColor = itemModel.get(borderColorAccessPath, true);
                 if (color != null) {
                     data.setItemVisual(idx, 'color', color);
                 }
+                if (borderColor != null) {
+                    data.setItemVisual(idx, 'borderColor', borderColor);
+                }
             };
 
```

Comment:
![image](https://user-images.githubusercontent.com/1956569/65424987-8b20cf80-de3f-11e9-91d4-46090ab6b4b4.png)

Why they are different

Reply:
The emptyCircle symbol doesn't have fillColor, so the strokeColor should be the same as color of corresponding series